@stellar/typescript-wallet-sdk/test/wallet.test.ts

```diff
@@ -1880,3 +1884,56 @@ describe("Http client", () => {
     expect(resp.data.transaction).toBeTruthy();
   });
 });
+
+describe("AuthHeaderSigner", () => {
+  it("full sep-10 auth using header token should work", async () => {
+    const wallet = Wallet.TestNet();
```

Comment:
You can also check that JWT from SEP-10 matches JWT issued by TS SDK
https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0010.md
(You can find private key in kotlin tests)

Reply:
err not sure what you mean here, could you explain further?

Reply:
I think you should run your code with the exact same parameters as in protocol examples and verify that token matches

Reply:
👍  added a test to check against the example token, and removed the `typ` header

https://github.com/stellar/typescript-wallet-sdk/pull/124/commits/ad7c0d2534ddb8e7b09f8df9c1d6db062b1e18a8

Reply:
Do you mind adding a test for custodial wallet too?